content/blog/joining-ggplot2/index.Rmd

```diff
@@ -0,0 +1,118 @@
+---
+output: hugodown::hugo_document
+
+slug: joining-ggplot2
+title: Joining the ggplot2 team
+date: 2025-01-09
+author: Teun van den Brand
+description: >
+    I joined the ggplot2 team and would like to share the experience.
+
+photo:
+  url: https://unsplash.com/photos/person-holding-green-plant-on-black-pot-CbZh3kaPxrE
+  author: Jonathan Kemper
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [other] 
+tags: []
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
```

Comment:
Can I get you to tick off the things you've completed so I now if they've been taken care of?